core/iwasm/common/wasm_application.c

```diff
@@ -105,6 +105,9 @@ wasm_application_execute_main(WASMModuleInstanceCommon *module_inst, int32 argc,
        the actual main function. Directly calling main function
        may cause exception thrown. */
     if ((func = wasm_runtime_lookup_wasi_start_function(module_inst))) {
+#if WASM_ENABLE_DEBUG_INTERP != 0
+        wasm_runtime_start_debug_instance(exec_env);
```

Comment:
Should check the return value?

Reply:
And after the following wasm_runtime_call_wasm, shall we destroy the debug instance?

Reply:
This API return the actual bind port number, or 0 if the debug instance not created. We don't check the return value here, because when user execute iwasm w/o `-g={ip}:{port}`, the API will return 0, but we still want the program to be executed

Reply:
> And after the following wasm_runtime_call_wasm, shall we destroy the debug instance?

We don't need to destroy the debug instance, it will be automatically destroyed when the corresponding cluster is destroyed.